File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.py
arr = list(map(int, input().split()))
ans = 0

#완전탐색 생각
#숫자는 100보다 작거나 같은 자연수
for i in range(1, 1000001):
    cnt = 0
    for j in arr:
        if i % j == 0:
            cnt += 1

    if cnt >= 3:
        ans = i
        break

print(ans)

File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
	public static void main(String args[]) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringTokenizer stk = new StringTokenizer(br.readLine(), " ");
		int arr[] = new int[5];
		for(int i = 0; i< 5; i ++) {
			arr[i] = Integer.parseInt(stk.nextToken());
		}
		
		for(int i = 1; i < 1000000; i++) {
			int cnt = 0;
			for(int j = 0; j< 5; j ++) {
				if(i % arr[j] == 0) {
					cnt +=1;
				}
			}
			if(cnt >= 3) {
				System.out.println(i);
				break;
			}
		}

	}
}
